### PR TITLE
NEW: Time measuring system

### PR DESCRIPTION
**Pull request by kornys**

### CODE REVIEW

**Comment by kornys** [comment]
Example of json output
```
{
  "io.enmasse.systemtest.brokered.clients.proton.python.MsgPatternsTest": {
    "testBasicMessage": {
      "TEST_EXECUTION": {
        "startTime": 1530093496105,
        "endTime": 1530093663416,
        "duration": 167311,
        "durationReadable": "00:02:47,311"
      },
      "CREATE_ADDRESS_SPACE-d0456472": {
        "startTime": 1530093497623,
        "endTime": 1530093536633,
        "duration": 39010,
        "durationReadable": "00:00:39,010"
      },
      "CREATE_USER-c4702cb6": {
        "startTime": 1530093537761,
        "endTime": 1530093542425,
        "duration": 4664,
        "durationReadable": "00:00:04,664"
      },
      "CREATE_USER-e1dc5bb4": {
        "startTime": 1530093542426,
        "endTime": 1530093545464,
        "duration": 3038,
        "durationReadable": "00:00:03,038"
      },
      "CREATE_ADDRESS-fbe2b996": {
        "startTime": 1530093545472,
        "endTime": 1530093651705,
        "duration": 106233,
        "durationReadable": "00:01:46,233"
      },
      "DELETE_ADDRESS-6d758148": {
        "startTime": 1530093654301,
        "endTime": 1530093663405,
        "duration": 9104,
        "durationReadable": "00:00:09,104"
      }
    }
  },
  "io.enmasse.systemtest.brokered.clients.rhea.MsgPatternsTest": {
    "testBasicMessage": {
      "TEST_EXECUTION": {
        "startTime": 1530093663439,
        "endTime": 1530093693945,
        "duration": 30506,
        "durationReadable": "00:00:30,506"
      },
      "CREATE_ADDRESS_SPACE-923a2b52": {
        "startTime": 1530093663441,
        "endTime": 1530093665489,
        "duration": 2048,
        "durationReadable": "00:00:02,048"
      },
      "CREATE_USER-0f765ac4": {
        "startTime": 1530093666576,
        "endTime": 1530093670086,
        "duration": 3510,
        "durationReadable": "00:00:03,510"
      },
      "CREATE_USER-72fe0d94": {
        "startTime": 1530093670087,
        "endTime": 1530093672907,
        "duration": 2820,
        "durationReadable": "00:00:02,820"
      },
      "CREATE_ADDRESS-51e5dfd8": {
        "startTime": 1530093672909,
        "endTime": 1530093682017,
        "duration": 9108,
        "durationReadable": "00:00:09,108"
      },
      "DELETE_ADDRESS-0d163cee": {
        "startTime": 1530093684839,
        "endTime": 1530093693938,
        "duration": 9099,
        "durationReadable": "00:00:09,099"
      }
    }
  }
}
```

File: systemtests/src/main/java/io/enmasse/systemtest/KeycloakClient.java [hunk 1]
```diff
@@ -259,6 +263,7 @@ public void deleteUser(String realm, String userName) throws Exception {
                 throw new RuntimeException(String.format("User {} is not deleted from keycloak", userName));
             }
         }
+        TimeMeasuringSystem.stopOperation(operationID);
```
Comment (lulf) [inline systemtests/src/main/java/io/enmasse/systemtest/KeycloakClient.java]: What happens if the exception is thrown and the stopOperation is never called? Just bad data? Consider wrapping it in a try{ } finally

Comment (kornys) [reply systemtests/src/main/java/io/enmasse/systemtest/KeycloakClient.java]: yeah just bad data 👍 good point I will update it

**Comment by kornys** [comment]
@lulf I moved calling stopOperation into finally block

**Comment by lulf** [comment]
@kornys I think this is an issue in more places than that, but I'm not sure if its worth sparkling the code with try {} finally everywhere, so I'm with with this.

**Comment by kornys** [comment]
@lulf yes you're right, but anyway when test failed and stopOperation is not called, it will only print wrong duration without affecting next tests. I will try to look at it more tomorrow.

**Pull request merged**